torchvision/prototype/datasets/_folder.py

```diff
@@ -54,7 +54,7 @@ def from_data_folder(
     dp: IterDataPipe = Filter(dp, functools.partial(_is_not_top_level_file, root=root))
     dp = hint_sharding(dp)
     dp = Shuffler(dp, buffer_size=INFINITE_BUFFER_SIZE)
-    dp = FileLoader(dp)
+    dp = FileOpener(dp)
```

Comment:
@pmeier I should mention that `FileLoader` has a default `mode="r"`, and `FileOpener` has a default `mode="b"`. We made that change to be consistent with other libraries. You may also want to specify the modes here as it fits your use cases.

Reply:
You have these mixed up, right? `FileLoader` had `mode="b"`, but `FileOpener` will now go with `mode="r"`, correct?

Reply:
Yep you're right. Sorry